loader_hub/docugami/base.py

```diff
@@ -0,0 +1,321 @@
+"""Docugami reader."""
+
+import io
+import os
+import re
+
+from lxml import etree
```

Comment:
would you be able to put this as a lazy import within the function ? otherwise tests will break

Reply:
sure!